README.md

```diff
@@ -38,6 +38,7 @@ The following authproc filter configuration options are supported:
   ```
   :warning: The configured Assurance Map array is merged recursively with the default one from above.
   * `defaultAssurance`: _Optional_, an array containing assurance value(s) to set by default when no assurance information is available. Defaults to `array()`, i.e. no assurance values are added by default.
+  * `minAssurance`: _Optional_, at least one of the Assurance values in the array MUST be present in user's state. Defaults to `null`, i.e. no assurance values are required.
```

Comment:
* `defaultAssurance`: _Optional_, an array containing assurance values to set by default when no assurance information is available or none of the required assurance values is present (see `minAssurance`). Defaults to `array()`, i.e. no assurance values are added by default.
* `minAssurance`: _Optional_, an array containing required assurance values. Defaults to `null`, i.e. no assurance values are required. When specified, at least one of the specified assurance values in the array MUST be present, otherwise the default assurance values will be assigned (see `defaultAssurance`).

Question: Why the `defaultAssurance` defaults to `array()` while `minAssurance` defaults to `null`?

Reply:
There is no specific reason. Do we need this be `array` for consistency?

Reply:
Yes. We should treat empty optional config parameters consistently (at least in the same module/filter)

Reply:
Done